tests/scripts/all.sh

```diff
@@ -1629,11 +1629,16 @@ component_test_full_cmake_clang () {
     msg "test: ssl-opt.sh default, ECJPAKE, SSL async (full config)" # ~ 1s
     tests/ssl-opt.sh -f 'Default\|ECJPAKE\|SSL async private'
 
+    msg "test: compat.sh default ciphers"
+    tests/compat.sh -m 'ssl3 tls1 tls1_1 tls12 dtls1 dtls12'
```

Comment:
Ok, what threw me off here was that we were doing non-default stuff but not the default stuff. I'm happy with doing no `compat.sh` here and keeping it only for the ASan component. IMO it would make sense to remove `ssl-opt.sh` here too for the same reason. (I mean it made sens to have `ssl-opt` here back when we didn't have `test_suite_ssl` doing complete handshakes, but now I don't think it does).